src/ui/public/saved_objects/saved_objects_client.js

```diff
@@ -126,10 +188,10 @@ export class SavedObjectsClient {
     return this._$http(options)
       .then(resp => get(resp, 'data'))
       .catch(resp => {
-        const respBody = resp.data || {};
+        const respBody = get(resp, 'data', {});
```

Comment:
I updated the whole file to use `_` instead of dereferencing the methods out to eliminate confusion.